py-polars/polars/expr/expr.py

```diff
@@ -4265,6 +4265,14 @@ def map_elements(
             This method is much slower than the native expressions API.
             Only use it if you cannot implement your logic otherwise.
 
+            For concreteness, suppose that the function is: `x ↦ sqrt(x)`.
+
+            For mapping elements of columns, consider:
+            `pl.col("colname").sqrt()`.
+
+            For mapping elements of inner lists, consider:
+            `pl.col("colname").list.eval(pl.element().sqrt())`.
```

Comment:
same comment as below but with `pl.col("col_name")` instead of `s`

Reply:
Applied this suggestion in the latest commit. Not sure about whitespace here: is it signigicant? I have aligned 'F' and '`'.

```
    Suppose that the function is: `x ↦ sqrt(x)`:

    - For mapping elements of a series, consider:
      `pl.col("col_name").sqrt()`.
    - For mapping inner elements of lists, consider:
      `pl.col("col_name").list.eval(pl.element().sqrt())`.
```

Reply:
> is it signigicant

I think so, but still haven't mastered the art of visually predicting whether sphinx docs are going to render correctly 😄 fancy running `make html` from the docs folder, and then posting a screenshot of these rendered docstrings?

Reply:
After the holidays, yes.

Reply:
![2024-05-01 23 22 58](https://github.com/pola-rs/polars/assets/7001793/352cb7a6-f195-4de2-8d59-9325aebce186)